src/Stryker.CLI/Stryker.CLI/StrykerCLI.cs

```diff
@@ -99,6 +103,11 @@ private void RunStryker(StrykerOptions options)
             PrintStykerASCIIName();
             _ = PrintStrykerVersionInformationAsync();
 
+            // Configure logging
+            ApplicationLogging.ConfigureLogger(options.LogOptions);
```

Comment:
Moving the logging configuration here means that whatever is used to bootstrap the stryker run (Stryker.CLI) now becomes responsible to set up logging that Stryker.Core uses. Our thoughts for Stryker.CLI are that it should basically only handle user interaction and is not actually responsible for any setup, Stryker.Core owns all it's configuration and does final validation.

How about we pass some 'initial logging' to RunMutationTest instead to allow bootstrappers to pass their logging onto stryker which stryker can then handle as it sees fit?

Reply:
The log messages are now passed into the stryker runner and from there into the `ApplicationLogging.ConfigureLogging` method.